v2/scripts/run_integration_tests.sh

```diff
@@ -64,7 +64,7 @@ printOutputAndExit() {
 }
 
 for ((i = 0; i < ${#STACK_CONFIGS[@]}; i++)); do
-    tsc --project tsconfig.json
+    npx tsc --project tsconfig.json
```

Comment:
Was this change needed in your side?

Reply:
yes we shouldn't assume tsc is installed globally